data/saas/config/rollbar_config.yml

```diff
@@ -0,0 +1,91 @@
+saas_config:
+  - fides_key: rollbar_connector
+    name: Rollbar SaaS Config
+    type: rollbar
+    description: A sample schema representing the Rollbar connector for Fidesops
+    version: 0.0.1
+
+    connector_params:
+      - name: domain
+      - name: read_access_token
+      - name: write_access_token
+      - name: page_size
+
+    client_config:
+      protocol: https
+      host: <domain>
+
+    test_request:
+      method: GET
+      path: /api/1/status/ping
+
+    endpoints:
+      - name: projects
+        requests:
+          read:
+            method: GET
+            path: /api/1/projects
+            headers:
+              - name: X-ROLLBAR-ACCESS-TOKEN
```

Comment:
Minor preference: I would use `X-Rollbar-Access-Token` instead of the uppercase version